test/automation/src/positron/positronPopups.ts

```diff
@@ -102,8 +102,16 @@ export class PositronPopups {
 	 * Can be called after a DropDownListBox is clicked. Selects the option with the given label.
 	 * @param label The label of the option to select.
 	 */
-	async clickOnModalDialogPopupOption(label: string) {
-		const el = this.code.driver.getLocator('.positron-modal-popup .positron-button .title').filter({ hasText: label });
+	async clickOnModalDialogPopupOption(label: string | RegExp) {
+		const el = this.code.driver.getLocator('.positron-modal-popup .positron-modal-popup-children').getByText(label);
```

Comment:
Might make sense to put `.positron-modal-popup .positron-modal-popup-children` up at the top of the file as an all uppercase constant